### ISSUE TITLE
Add support for import .. from ..

### ISSUE THREAD

**Issue by mehcode** [capaj/npm-autocomplete#2]
The package works great. Have you thought about adding support for `import ... from ...`?

**Comment by capaj** [comment]
yes, I have thought about it, will try adding it soon. Shouldn't be too hard.
